x-pack/plugins/ingest_pipelines/public/application/sections/pipelines_list/main.tsx

```diff
@@ -79,6 +83,37 @@ export const PipelinesList: React.FunctionComponent<RouteComponentProps> = ({
     redirectToPathOrRedirectPath(getListPath());
   };
 
+  const isManagedPipelines = (allPipelines: Pipeline[], isManaged: boolean) => {
+    return allPipelines.filter((pipeline) => {
+      return isManaged ? pipeline.isManaged : !pipeline.isManaged;
+    });
```

Comment:
How do you feel about simplifying this logic a bit? Perhaps something like this might be easier to read:

```suggestion
     return allPipelines.filter((pipeline) => pipeline.isManaged === isManaged);
```